.gitignore

```diff
@@ -1,7 +1,7 @@
 *.pyc
 *.egg-info
 .tox
-.coverage*
+.coverage
```

Comment:
there are temporary .coverage-... files created while running tests, I think this is what this line was for